test/unit/vl-map-search.test.html

```diff
@@ -89,6 +89,38 @@
         });
       });
 
+      test('wanneer de gebruiker een Lambert coördinaat zoekt zullen bij geopunt de locaties opgehaald worden', () => {
+        const value = '104719.27, 192387.25';
+
+        const response = new Promise((resolve, reject) => {
+          resolve(new Response(JSON.stringify({}), {status: 200}));
+        });
+
+        const fetchStub = sandbox.stub(window, 'fetch');
+        const xy = value.replace(/\s/g, '');
+        fetchStub.withArgs('https://loc.geopunt.be/v4/Location?xy=' + xy + '&c=5').returns(response);
+
+        const element = fixture('vl-map-with-search-fixture');
+        return customElements.whenDefined('vl-map').then(() => {
+          const selectElement = element.shadowRoot.querySelector('vl-map-search').shadowRoot.querySelector('select');
+
+          selectElement.dispatchEvent(new CustomEvent('search', {
+            detail: {
+              value: value,
+            },
+          }));
+
+          return new Promise((resolve) => {
+            (async () => {
+              while (!fetchStub.called) {
+                await new Promise((resolve) => setTimeout(resolve, 100));
+              }
+              resolve();
+            })();
+          });
+        });
+      });
+
       test('wanneer de gebruiker een suggestie selecteert zal bij geopunt de locatie opgehaald worden', () => {
```

Comment:
lege test?

Reply:
net even gecontroleerd maar zowel in mijn IDE als op de origin van deze branch lijkt de test aanwezig. Zie jij de code hiervan niet?

Reply:
Nee ... (of ik kijk scheef)

Reply:
Ik zie de content ook wel. Je moet de code expanden in de code review (of eens de raw file bekijken op de branch).